modules/ROOT/pages/about-classloading-isolation.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Mule Runtime engine is developed using Java and many other third party libraries. Mule apps can also contain other Java libraries, particularly extensions that provide integration functionality, such as the HTTP connector. These extensions can also contain other JAR libraries.
+The Mule Runtime engine is developed using Java and many other third party libraries. Mule apps can also contain other Java libraries, particularly modules that extend the runtime to provide integration functionality, such as the HTTP connector. These extensions can also contain other JAR libraries.
```

Comment:
Shouldn't the second `extensions` also be changed? Also, I would add two examples to make it clearer: `the HTTP connector and the XML module`.

Reply:
Added XML module. Instead of changing "extensions" throughout the page, I'm hoping that "modules that extend" with the examples will be enough.